spec/expectations/should_spec.rb

```diff
@@ -5,7 +5,7 @@
   before :all do
     path = RbConfig::CONFIG['bindir']
     exe  = RbConfig::CONFIG['ruby_install_name']
-    file = File.dirname(__FILE__) + '/should.rb'
+    file = File.dirname(__FILE__) + '/should.rb 2>&1'
```

Comment:
It prevents running MSpec specs on Windows. Not sure whether it's important.

It would be a little more difficult to check STDERR of the command in this test. Another option is to write a deprecation message to STDOUT instead of STDERR.

Reply:
I think `2>&1` might actually work on Windows, could you move it to the line below though, so `file` is still an actual file?